src/core/directives/metadata_directives.ts

```diff
@@ -1225,3 +1225,38 @@ export class HostBindingMetadata {
 export class HostListenerMetadata {
   constructor(public eventName: string, public args?: string[]) {}
 }
+
+/**
+ * Interface for creating NgModuleMetadata
+ */
+export interface NgModuleMetadataType {
```

Comment:
Well, basically everything has moved, been removed or been rewritten in ng2: I'm not exaggerating 😁. Seems a bit pointless to have one TODO comment in there, but you're the boss 😉